bridges/otelzap/core_test.go

```diff
@@ -31,6 +33,30 @@ func TestCore(t *testing.T) {
 	assert.Equal(t, log.SeverityInfo, got.Severity())
 }
 
+func TestCoreEnabled(t *testing.T) {
```

Comment:
What about `logger.Debug(testMessage)`. Will it be recorded or not? I think it deserves a test case.